sys-apps/pciutils/patches/pciutils-3.6.0.patchset

```diff
@@ -0,0 +1,24 @@
+From 137958092e6ca0b5eb293d9a8f4e7e6cf5a857ad Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Sun, 1 Jul 2018 18:42:31 +0200
+Subject: Build fix
+
+
+diff --git a/lib/sysdep.h b/lib/sysdep.h
+index e525dc4..3d7944d 100644
+--- a/lib/sysdep.h
++++ b/lib/sysdep.h
+@@ -68,6 +68,10 @@ typedef u16 word;
+ #endif
+ #endif
+ 
++#ifdef PCI_OS_HAIKU
++#include <endian.h>
++#endif
++
+ #ifdef PCI_OS_SYLIXOS
```

Comment:
Can't you use #if defined(PCI_OS_SYLIXOS) || defined(PCI_OS_HAIKU)?

Reply:
Each OS has its dedicated `#ifdef` bloc, so it is better to keep the one for Haiku in a separate bloc too.